src/node.h

```diff
@@ -562,13 +562,15 @@ extern "C" NODE_EXTERN void node_module_register(void* mod);
 /* Called after the event loop exits but before the VM is disposed.
  * Callbacks are run in reverse order of registration, i.e. newest first.
  */
-NODE_EXTERN void AtExit(void (*cb)(void* arg), void* arg = 0);
+NODE_EXTERN void AtExit(void (*cb)(void* arg), void* arg = nullptr);
```

Comment:
Anyone know is this has any chance of effecting the ecosystem?

Reply:
We already use `nullptr` elsewhere in this file, so I think this should be okay.